scripts/build.sh

```diff
@@ -13,6 +13,7 @@ fi
 GOBUILD_TAGS="novirt noaugeas "
 VERSION=$($(dirname $0)/pkg-version --full)
 LDFLAGS="-X github.com/gluster/glusterd2/gdctx.GlusterdVersion=$VERSION"
+LDFLAGS+=" -B 0x$(head -c20 /dev/urandom | od -An -tx1 | tr -d ' \n')"
```

Comment:
AFAICT, this would make the builds un-reproducible. As a result, maybe best to avoid doing this. More info here: https://reproducible-builds.org/